app/models/broken_window/metric.rb

```diff
@@ -6,7 +6,7 @@ class Metric < ActiveRecord::Base
 
     SNOOZE_INTERVAL = 2.weeks
 
-    has_many :measurements
+    has_many :measurements, -> { order("created_at ASC, id ASC") }
```

Comment:
usage of unsanitized strings is deprecated. `order(created_at: :asc, id: :asc)`

Reply:
why is order being added here? it's like a default scope, you will now have this order everywhere. If this is needed to show the graph, then putting the order on the graph controller seems better to me